check_elasticsearch_snapshot

```diff
@@ -57,12 +57,16 @@ THRESHOLD_SUFFIX_MULTIPLIERS = {
 
 
 def query(args, qry):
-    req = requests.get('{}://{}:{}/{}{}'.format(args.scheme,
-                                                args.server,
-                                                args.port,
-                                                args.subfolder,
-                                                qry),
-                       auth=(args.username, args.password))
+    try:
+        req = requests.get('{}://{}:{}/{}{}'.format(args.scheme,
+                                                    args.server,
+                                                    args.port,
+                                                    args.subfolder,
+                                                    qry),
+                           auth=(args.username, args.password))
+    except requests.exceptions.ConnectionError as exception:
```

Comment:
Let's call this conn_error or something more specific, in case we want to handle others in the future